docs/Users_Guide/glossary.rst

```diff
@@ -5445,6 +5445,11 @@ METplus Configuration Glossary
 
      | *Used by:*  EnsembleStat
 
+   ENSEMBLE_STAT_CLIMO_CDF_DIRECT_PROB
```

Comment:
@georgemccabe I did NOT add climo_cdf.direct_prob to the default Ensemble-Stat config file. As the logic exists today, it does not actually APPLY to Ensemble-Stat. But for convenience, I used the same "climo_cdf" setup across all the tools. And work on https://github.com/dtcenter/MET/issues/1602 might make it applicable. I haven't through through all the details yet though.

I recommend leaving this as-is and am not recommending changes. Just wanted to clarify.